WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -179,31 +295,40 @@ private extension ZendeskUtils {
         return (allBlogs.map { $0.logDescription() }).joined(separator: Constants.blogSeperator)
     }
 
-    func getTags() -> [String] {
-
-        var tags = [String]()
+    static func getTags() -> [String] {
 
         let context = ContextManager.sharedInstance().mainContext
-        let accountService = AccountService.init(managedObjectContext: context)
+        let blogService = BlogService(managedObjectContext: context)
 
-        if let defaultAccount = accountService.defaultWordPressComAccount() {
-
-            tags = defaultAccount.blogs.filter {
-                $0.planTitle != nil
-                }.map {
-                    $0.planTitle!
-                }.unique
+        // If there are no sites, then the user has an empty WP account.
+        guard let allBlogs = blogService.blogsForAllAccounts() as? [Blog], allBlogs.count > 0 else {
+            return [Constants.wpComTag]
+        }
 
-            tags.append(Constants.wpComTag)
+        // Get all unique site plans
+        var tags = allBlogs.filter {
+            $0.planTitle != nil
+            }.map {
+                $0.planTitle!
+            }.unique
 
-        } else {
+        // If any of the sites have jetpack installed, add jetpack tag.
+        let jetpackInstalled = allBlogs.map { $0.jetpack?.isInstalled }
+        if jetpackInstalled.contains(true) {
```

Comment:
You can probably simplify this snippet like this:

```
let jetpackBlog = allBlogs.first { $0.jetapck?.isInstalled == true }
if let _ = jetpackBlog {
   // ..
}
```

This way you both, stop iterating thru the blogs whenever one of them is `jetpack enabled`, and avoid checking for `true` values into the array (the variable should either be nil, or have the first jetpack enabled blog).